pkg/jobs/template/node-collector.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: node-collector
+spec:
+  template:
+    metadata:
```

Comment:
can we add more specific labels here? To show that this image is created by trivy, etc.